sa/sa.go

```diff
@@ -1602,10 +1603,14 @@ func (ssa *SQLStorageAuthority) GetOrder(ctx context.Context, req *sapb.OrderReq
 	if req.UseV2Authorizations != nil {
 		useV2Authzs = *req.UseV2Authorizations
 	}
+	useV2Authzs = useV2Authzs || features.Enabled(features.DisableAuthz2Orders)
```

Comment:
Can you add a comment to this line explaining the logical or with the disabled flag? I was able to piece together what's going on here from the PR desc and context but I think if someone looked at this code in a few week's time it wouldn't be super intuitive why `useV2Authzs` would be set true here when the disabled feature flag is enabled.